app/views/idv/phone_errors/failure.html.erb

```diff
@@ -4,16 +4,16 @@
       heading: t('idv.failure.phone.rate_limited.heading'),
       current_step: :verify_phone_or_address,
       options: [
-        decorated_session.sp_name && {
-          url: return_to_sp_failure_to_proof_path(step: 'phone', location: request.params[:action]),
-          text: t('idv.troubleshooting.options.get_help_at_sp', sp_name: decorated_session.sp_name),
-          new_tab: true,
-        },
         {
           url: MarketingSite.contact_url,
           text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
+        decorated_session.sp_name && {
```

Comment:
hi @jmax-gsa ... looks like the Cancel link is supposed to be at [the bottom of the page](https://cm-jira.usa.gov/secure/attachment/396983/screenshot-1.png) and not in the options section.

Reply:
Moved.